### PR TITLE
mediatek: filogic: rework 02_network setup

### PR DESCRIPTION
**Pull request by aiamadeus**
- Remove redundant code for tuf-ax4200
- Fix network config for mt7986a/b-rfb
- Move the mac address setting together

### CODE REVIEW

File: target/linux/mediatek/filogic/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -9,16 +9,16 @@ mediatek_setup_interfaces()
 
 	case $board in
 	asus,tuf-ax4200)
-		CI_UBIPART="UBI_DEV"
```
Comment (Ansuel) [inline target/linux/mediatek/filogic/base-files/etc/board.d/02_network:12]: Any idea why this was there from the start?

**Comment by Ansuel** [comment]
@aiamadeus also i would split this in 3 commit one for each topic.

File: target/linux/mediatek/files-5.15/arch/arm64/boot/dts/mediatek/mt7986a-rfb.dtsi [hunk 2]
```diff
@@ -337,7 +337,7 @@
 
 		port@4 {
 			reg = <4>;
-			label = "wan";
+			label = "lan5";
```
[the remark this answers is not in the document]
Comment (Ansuel) [reply target/linux/mediatek/files-5.15/arch/arm64/boot/dts/mediatek/mt7986a-rfb.dtsi:340]: Thanks for the clarification. Then I'm ok with the change! Then I guess the only change is to split this in 3 commit.

File: target/linux/mediatek/filogic/base-files/etc/board.d/02_network (hunk 1 above)
Comment (aiamadeus) [reply target/linux/mediatek/filogic/base-files/etc/board.d/02_network:12]:
```
	asus,tuf-ax4200)
		CI_UBIPART="UBI_DEV"
		addr=$(mtd_get_mac_binary_ubi "Factory" 0x4)
```
I think this is just a simple copy from other places.

File: target/linux/mediatek/files-5.15/arch/arm64/boot/dts/mediatek/mt7986a-rfb.dtsi (hunk 2 above)
[the remark this answers is not in the document]
Comment (nbd168) [reply target/linux/mediatek/files-5.15/arch/arm64/boot/dts/mediatek/mt7986a-rfb.dtsi:340]: I don't agree with this change. This port has been set aside from the regular LAN ports to mirror a typical LAN/WAN port setup to be used on MTK customer boards. I'd prefer to leave it as-is.

**Pull request merged**